### ISSUE TITLE
Error when signing up initial user.

### ISSUE THREAD

**Issue by steve-bate** [blitz-js/blitz#2461]
### What is the problem?

Saw an error when trying the Blitz tutorial. I was able to install Blitz, create a tutorial project, and display the initial Blitz page. When doing the next step, signing up a new user, I saw an error in the console.

### Paste all your error logs here:

```
c:\ws\src\node_http_parser.cc:567: Assertion `args[3]->IsInt32()' failed.
 1: 00007FF7D78386BF v8::internal::Isolate::ArchiveSpacePerThread+4719
 2: 00007FF7D77CB246 v8::internal::wasm::WasmCode::safepoint_table_offset+67414
 3: 00007FF7D77CB5C1 v8::internal::wasm::WasmCode::safepoint_table_offset+68305
 4: 00007FF7D77B0543 v8::base::CPU::has_sse+14915
 5: 00007FF7D80F3969 v8::internal::Builtins::builtin_handle+318073
 6: 00007FF7D80F2F01 v8::internal::Builtins::builtin_handle+315409
 7: 00007FF7D80F31F1 v8::internal::Builtins::builtin_handle+316161
 8: 00007FF7D80F3033 v8::internal::Builtins::builtin_handle+315715
 9: 00007FF7D81C98B1 v8::internal::SetupIsolateDelegate::SetupHeap+455217
10: 00007FF7D81639B9 v8::internal::SetupIsolateDelegate::SetupHeap+37689
11: 00007FF7D8160DFF v8::internal::SetupIsolateDelegate::SetupHeap+26495
12: 00007FF7D82553DA v8::internal::SetupIsolateDelegate::SetupHeap+1027418
13: 00007FF7D81639B9 v8::internal::SetupIsolateDelegate::SetupHeap+37689
14: 00007FF7D81639B9 v8::internal::SetupIsolateDelegate::SetupHeap+37689
15: 00007FF7D81639B9 v8::internal::SetupIsolateDelegate::SetupHeap+37689
16: 00007FF7D81639B9 v8::internal::SetupIsolateDelegate::SetupHeap+37689
17: 00007FF7D81639B9 v8::internal::SetupIsolateDelegate::SetupHeap+37689
18: 00007FF7D81639B9 v8::internal::SetupIsolateDelegate::SetupHeap+37689
19: 00007FF7D81639B9 v8::internal::SetupIsolateDelegate::SetupHeap+37689
20: 00007FF7D820FA48 v8::internal::SetupIsolateDelegate::SetupHeap+742344
21: 00007FF7D8160FCC v8::internal::SetupIsolateDelegate::SetupHeap+26956
22: 00007FF7D82553DA v8::internal::SetupIsolateDelegate::SetupHeap+1027418
23: 00007FF7D81639B9 v8::internal::SetupIsolateDelegate::SetupHeap+37689
24: 00007FF7D81639B9 v8::internal::SetupIsolateDelegate::SetupHeap+37689
25: 00007FF7D818FCB3 v8::internal::SetupIsolateDelegate::SetupHeap+218675
26: 00007FF7D82113A5 v8::internal::SetupIsolateDelegate::SetupHeap+748837
27: 00007FF7D8183513 v8::internal::SetupIsolateDelegate::SetupHeap+167571
28: 00007FF7D8161B0B v8::internal::SetupIsolateDelegate::SetupHeap+29835
29: 00007FF7D801A0E3 v8::internal::Execution::CallWasm+1891
30: 00007FF7D801A1EB v8::internal::Execution::CallWasm+2155
31: 00007FF7D801AC5A v8::internal::Execution::TryCall+378
32: 00007FF7D7FF7945 v8::internal::MicrotaskQueue::RunMicrotasks+501
33: 00007FF7D7FF76A0 v8::internal::MicrotaskQueue::PerformCheckpoint+32
34: 00007FF7D81659FF v8::internal::SetupIsolateDelegate::SetupHeap+45951
35: 0000030EBFA93C85
```

### Paste all relevant code snippets here:

N/A

### What are detailed steps to reproduce this?

1. Install Blitz
2. Run the development server
3. Sign up a new user

### Run `blitz -v` and paste the output here:

```
Windows 10 | win32-x64 | Node: v16.0.0

blitz: 0.36.4 (global)
blitz: 0.36.4 (local)

  Package manager: yarn
  System:
    OS: Windows 10 10.0.19041
    CPU: (4) x64 Intel(R) Core(TM) i5-4570 CPU @ 3.20GHz
    Memory: 13.89 GB / 23.94 GB
  Binaries:
    Node: 16.0.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.4 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 7.12.1 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  npmPackages:
    @prisma/client: ~2.20 => 2.20.1
    blitz: 0.36.4 => 0.36.4
    prisma: ~2.20 => 2.20.1
    react: 0.0.0-experimental-6a589ad71 => 0.0.0-experimental-6a589ad71
    react-dom: 0.0.0-experimental-6a589ad71 => 0.0.0-experimental-6a589ad71
    typescript: ~4.2 => 4.2.4
```

### Please include below any other applicable logs and screenshots that show your problem:

N/A

**Comment by flybayer** [comment]
Hey! Solution is to upgrade `prisma` and `@prisma/client` to `2.23.1

**Comment by steve-bate** [comment]
Thanks! That fixed the problem (2.24.1 rather than 2.23.1, which didn't exist).
